### PR TITLE
Assessments/2022 12 02 booleans

### PR DESCRIPTION
**Pull request by jackypop13**

### CODE REVIEW

File: assessments/2022-12-02-booleans/code.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const needHeavyCoat=(goingOutside,somewhereCold)=>{
+  return goingOutside && somewhereCold
```
Comment (gigamonkey) [inline assessments/2022-12-02-booleans/code.js:2]: Good.

File: assessments/2022-12-02-booleans/code.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+}
+const needSunscreen=(goingToBeach,goingSkiing)=>{
+  return goingToBeach || goingSkiing
```
Comment (gigamonkey) [inline assessments/2022-12-02-booleans/code.js:5]: Yup.

File: assessments/2022-12-02-booleans/code.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+}
+const needMittens=(goingOutside,somewhereWarm)=>{
+  return goingOutside || !somewhereWarm
```
Comment (gigamonkey) [inline assessments/2022-12-02-booleans/code.js:8]: Yup.

File: assessments/2022-12-02-booleans/code.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+}
+const isVenomous=(striped,blueHead)=>{
+  if (!striped && blueHead){
+  return false}
+  return true
```
Comment (gigamonkey) [inline assessments/2022-12-02-booleans/code.js:13]: This is correct but formatted weirdly which makes it harder than necessary to read. At least to people who are used to code formatted in a standard way. You can use the right-click "Format document" feature in the editor to clean up a lot of things and start getting a sense of how code should be formatted. In this case it should be something like:

```javascript
const isVenomous = (striped, blueHead) => {
  if (!striped && blueHead) {
    return false
  }
  return true
}
```

Though the formatting issues can be avoided by simplifying this code. You should watch this video I made last year for the IB CS kids for some ideas. https://www.youtube.com/watch?v=yGX-OGhyf4Q

**Review by gigamonkey — APPROVED** [review]
Left some comments but you can go ahead and merge this.

**Comment by jackypop13** [comment]
should i merge

**Comment by jackypop13** [comment]
never mind i did not see what you said at the top im merging this

**Pull request merged**